crates/extensions/c8y_auth_proxy/src/server.rs

```diff
@@ -841,7 +841,7 @@ mod tests {
     #[tokio::test]
     async fn forwards_successful_responses() {
         let _ = env_logger::try_init();
-        let mut server = mockito::Server::new();
+        let mut server = mockito::Server::new_async().await;
         let _mock = server
```

Comment:
Don't we need to change from `crete()` to `create_async()` in this file?

Reply:
Looking at the crate documentation, there doesn't seem to be much difference between the two, i.e. both will work here, but using `.create_async` we don't block and await, which would maybe have performance implications when registering many mocks, but in our use case there's no difference as long as `.create` also works

Reply:
Thanks @Bravo555 for the insight. However, I decided to apply @rina23q suggestions to be consistent.